test/testcafe/framework/page-objects/EnduserConsentPageObject.js

```diff
@@ -13,15 +14,31 @@ export default class EnduserConsentPageObject extends ConsentPageObject {
     return this.form.clickCancelButton();
   }
 
-  getInfoItemTexts() {
+  getInfoItemTextBrowser() {
+    if(userVariables.v3) {
+      return this.form.getElement('[data-se="text-browser"]').innerText;
+    }
+    return this.form.getInnerTexts('.enduser-email-consent--info');
+  }
+
+  getInfoItemTextAppName() {
+    if(userVariables.v3) {
+      return this.form.getElement('[data-se="text-appName"]').innerText;
+    }
     return this.form.getInnerTexts('.enduser-email-consent--info');
   }
 
   getSaveButtonLabel() {
+    if (userVariables.v3) {
+      return this.form.getElement('[data-type="save"]').innerText; 
+    }
     return this.form.getElement('input[type="submit"]').value;
   }
 
   getCancelButtonLabel() {
+    if (userVariables.v3) {
+      return this.form.getElement('[data-type="cancel"]').innerText; 
+    }
     return this.form.getElement('input[data-type="cancel"]').value;
```

Comment:
Can you use `this.form.getButton('Cancel')` and then divide the behavior on `innerText` vs `value` if necessary after that? This applies to `getSaveButtonLabel` too

Reply:
Actually, both of these `get*ButtonLabel` methods already exist on BaseFormObject and on BasePageObject there is already `getSaveButtonLabel`

Reply:
Can you update to use `this.form.getCancelButtonLabel()`?